packages/stores/types.ts

```diff
@@ -258,6 +262,14 @@ type TTradersHubStore = {
     is_eu_user: boolean;
     is_real: boolean;
     selectRegion: (region: string) => void;
+    selected_account_type: string;
+    platform_real_balance: { balance: number; currency: string };
```

Comment:
@shayan-deriv Please update the types correctly and remove those that have been removed 🙏🏻

Reply:
done. please check again